src/Services/WidgetDocumentQuery.php

```diff
@@ -2,114 +2,112 @@
 
 namespace Bonnier\WP\Cxense\Services;
 
+use Bonnier\WP\Cxense\Http\HttpRequest;
+use Bonnier\WP\Cxense\Exceptions\HttpException;
+use Bonnier\WP\Cxense\Exceptions\WidgetException;
+use Bonnier\WP\Cxense\Exceptions\WidgetMissingId;
+use Bonnier\WP\Cxense\Parsers\Document;
 use Bonnier\WP\Cxense\WpCxense;
 
 class WidgetDocumentQuery
 {
-    private $widgetId;
-    private $siteId;
     private $cxenseUserId;
     private $matchingMode;
-    private $categories;
-    private $tags;
-    private $query;
-    private $context;
+    private $arrPayload = [];
 
     const POPULAR = 'trend';
     const RELATED = 'contextual'; // Articles similar to the current article.
     const SIMILAR_READS = 'collabctx'; // People who have read the current article have also read these articles.
     const RECENTLY_VIEWED = 'recent';
 
+    /**
+     * WidgetDocumentQuery constructor.
+     */
     public function __construct()
     {
-        $this->setWidgetId(WpCxense::instance()->settings->getSortbyWidgetId());
-        $this->setSiteId(WpCxense::instance()->settings->getSiteId());
-    }
-
-    public static function make()
-    {
-        return new static();
+        $this->validateWidgetId(WpCxense::instance()->settings->get_setting_value('sortby_widget_id', get_locale()));
+        $this->setSiteId(WpCxense::instance()->settings->get_setting_value("site_id", get_locale()));
```

Comment:
You cannot access `get_setting_value`, so as I wrote in my previous comment, you need to use `WpCxense::instance()->settings->getSortbyWidgetId()` and `WpCxense::instance()->settings->getSiteId()`